ca_gazebo/scripts/publishtagspose.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python  
+import roslib
+import rospy
+import math
+import tf
+import geometry_msgs.msg
+import turtlesim.srv
```

Comment:
I guess you don't need `turtlesim.srv` for what you're doing. Check other imports too.

Reply:
Adressed